### PR TITLE
LA-380, Nessie .ebextensions (SSL cert coming soon)

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/LA-380

I'd like to get a jump on peer review. The TODO item: `.ebextensions/10_install_httpd.config` > nessie_openssl_x509.crt

### CODE REVIEW

**Comment by paulkerschen** [comment]
Looks like a well-ordered bunch of .ebextensions to me, @johncrossman. Maybe @sandeepmjay also wants to glance over, as someone who's been wrestling such things lately?

**Comment by sandeepmjay** [comment]
LGTM @johncrossman.

File: .ebextensions/00_ami.config [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    gcc-c++: []
+    git: []
+    mod24_ssl: []
+    openssl-devel: []
+
+option_settings:
+  aws:autoscaling:launchconfiguration:
+    EC2KeyName: cloudlrs-aws-eb
+    InstanceType: t2.small
+    IamInstanceProfile: aws-elasticbeanstalk-ec2-role
```
Comment (sandeepmjay) [inline .ebextensions/00_ami.config]: @johncrossman Nessi needs to have this security group config to SSH in custom VPC    
SSHSourceRestriction: tcp, 22, 22, sg-8ece7af1

File: .ebextensions/00_ami.config [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+ # Custom load balancer listener (port 433)
+  aws:elbv2:listener:443:
+    ListenerEnabled: 'true'
+    Protocol: HTTPS
+    SSLCertificateArns:
+
+  # Load Balancer security group
+  aws:elbv2:loadbalancer:
+    SecurityGroups:
```
Comment (sandeepmjay) [inline .ebextensions/00_ami.config]: @johncrossman  data-loch uses these security groups

    SecurityGroups: [sg-40c5713f]
    ManagedSecurityGroup: sg-40c5713f

**Comment by johncrossman** [comment]
Thank you, @sandeepmjay  Amendments in.

**Comment by sandeepmjay** [comment]
Merging when Mr.T green lights it. Thanks @johncrossman.

**Pull request merged**